dressup/src/components/Categories/Categories.js

```diff
@@ -0,0 +1,14 @@
+import "./Categories.css";
+
+function Categories() {
+  return (
+    <div className="categories">
+      <p>Men</p>
+      <p>Women</p>
+      <p>Kids</p>
+      <p>Accessories</p>
+    </div>
+  );
+}
```

Comment:
You can use ES6+ syntax here

Reply:
What ES6 syntax exactly . I'm not able to figure out? @amantulsyan35